docs/shared/migration/angular-integrated.md

```diff
@@ -0,0 +1,103 @@
+# Migrating an Angular CLI workspace to an Integrated Nx Monorepo
+
+To take advantage of Nx's monorepo features provided by Nx and the Nx Angular plugin, you can also perform a migration from an Angular CLI to an Integrated Nx Monorepo with the command:
+
+```shell
+ng add @nrwl/angular@<version_number>
+```
+
+**Note**: To migrate to legacy versions of Nx prior to Nx 13.10, run the command:
+
+```shell
+ng add @nrwl/workspace@<version_number>
+```
+
+**Note**: Refer to the [Nx and Angular Version Compatibility Matrix](/packages/angular/documents/angular-nx-version-matrix) for matching Angular and Nx versions.
+
+**Note**: Support for workspaces with multiple applications and libraries was added in Nx v14.1.0. If you are migrating using an older version of Nx, your workspace can only contain one application and no libraries in order to use the automated migration, otherwise, you can still [migrate manually](#transitioning-manually).
+
+> The automated migration supports Angular CLI workspaces with a standard structure, configurations and features. If your workspace has deviated from what the Angular CLI generates, you might not be able to use the automated migration and you will need to [manually migrate your workspace](#transitioning-manually).
+>
+> Currently, the automated migration supports workspaces using the following executors (builders):
+>
+> - `@angular-devkit/build-angular:browser`
+> - `@angular-devkit/build-angular:dev-server`
+> - `@angular-devkit/build-angular:extract-i18n`
+> - `@angular-devkit/build-angular:karma`
+> - `@angular-devkit/build-angular:ng-packagr`
+> - `@angular-devkit/build-angular:protractor`
+> - `@angular-devkit/build-angular:server`
+> - `@angular-eslint/builder:lint`
+> - `@cypress/schematic:cypress`
+> - `@nguniversal/builders:prerender`
+> - `@nguniversal/builders:ssr-dev-server`
+>
+> Support for other executors may be added in the future.
+
+This installs the `@nrwl/angular` (or `@nrwl/workspace`) package into your workspace and runs a generator (or schematic) to transform your workspace. The generator applies the following changes to your workspace:
+
+- Installs the `nx` and `@nrwl/workspace` packages.
```

Comment:
These feel like a lot. Can we shorten these or remove them entirely? I feel like I just look at the workspace file structure below it anyway. That gives me all the info I need already.